web/src/drafts.js

```diff
@@ -617,20 +617,20 @@ export function launch() {
     }
 
     function setup_event_handlers() {
-        $(".restore-draft").on("click", function (e) {
+        $(".restore-overlay-message").on("click", function (e) {
```

Comment:
This change is potentially wrong: `restore-overlay-message` appears in the scheduled messages overlay too. The `$("#drafts_table ...")` approach used elsewhere seems like an appropriate fix.

Please carefully audit this whole commit for this class of issue.

Reply:
@sbansal1999  I think in this commit, instead of replacing `draft` with `overlay-message`, we can replace them with `.draft-row .overlay-message`.  This would ensure everything works correctly.

> This resulted in the same styles being applied twice.

@sbansal1999  Can you describe what styles you are talking about?

Reply:
I feel what Tim has suggested feels like a more intuitive way to me as it would clearly tell that the element belongs to the drafts table. I would love to hear your thoughts on this one @amanagr.

> Can you describe what styles you are talking about?

I am talking about the styles that were removed in the next commit: b653d1e46616b657e2f45f00f00279feeb9ae838

Reply:
I think either option is workable here.